api/src/main/resources/application-cloud.properties

```diff
@@ -15,6 +15,4 @@
 # limitations under the License.
 #
 
-spring.h2.console.enabled=false
-spring.flyway.locations=classpath:db/mysql
```

Comment:
This looks good but I'd love to know why

Reply:
It was removed because it was already provided in application.properties so the spring.flyway.locations was redundant here.